test/grouping.jl

```diff
@@ -171,18 +171,18 @@ module TestGrouping
     @test gd[4] == DataFrame(Key1="B", Key2="B", Value=4)
 
     @testset "grouping with missings" begin
-        df = DataFrame(Key1 = ["A", missing, "B", "B", "A"],
-                       Key2 = CategoricalArray(["B", "A", "A", missing, "A"]),
-                       Value = 1:5)
+        global df = DataFrame(Key1 = ["A", missing, "B", "B", "A"],
```

Comment:
I don't think this is right. `df` shouldn't be used outside of this testset.

Reply:
The problem is the name.  Several other `testset`s use a `DataFrame` called `df` which is defined at the module level.  I know this dataframe is not used outside this testset but I am just trying to silence the warning.  In another file I did that by changing the name `df` -> `df1` in the testset and, in your review, you said not to do that.  In this case I added the `global` and you say not to do that.  What do you want done?

Reply:
Nothing. ;-)

I don't care about these warnings, they will go away in 1.0 anyway. It would be too bad if they prompted us to make our code worse.